packages/web/index.js

```diff
@@ -155,5 +151,16 @@ module.exports = (neutrino, opts = {}) => {
         }]
       });
       config.output.filename('[name].[chunkhash].js');
+    })
+    .when(process.env.NODE_ENV === 'production', () => {
+      neutrino.use(chunk);
+      neutrino.use(minify);
+
+      if (options.imagemin) {
```

Comment:
Is the goal to switch this to `neutrino.options.optimize` when it's available?

Reply:
Yes, but also to pass options if desired.